src/network/connection.cpp

```diff
@@ -64,7 +64,7 @@ BufferedPacketPtr makePacket(Address &address, const SharedBuffer<u8> &data,
 {
 	u32 packet_size = data.getSize() + BASE_HEADER_SIZE;
 
-	BufferedPacketPtr p(new BufferedPacket(packet_size));
+	auto p = std::make_shared<BufferedPacket>(packet_size);
```

Comment:
`BufferedPacketPtr` makes it more readable. No need to change this.

Reply:
@SmallJoker,
https://github.com/ros-planning/moveit/issues/1411
https://stackoverflow.com/questions/20895648/difference-in-make-shared-and-normal-shared-ptr-in-c